tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -161,49 +158,78 @@ jobs:
         commitId: $(OnnxRuntimeGitCommitHash)
     - template: templates/clean-agent-build-directory-step.yml
 
-- job: Windows_Packaging_GPU_x64
+
+- job: Windows_Packaging_GPU
   workspace:
     clean: all
-  timeoutInMinutes:  120    
-  pool: 'Win-GPU-CUDA10'
+  pool: 'Win-GPU-2019'
+  timeoutInMinutes:  120
   variables:
-    buildDirectory: '$(Build.BinariesDirectory)'
-    buildConfig: 'RelWithDebInfo'
-    buildArch: 'x64'
-
+    EnvSetupScript: setup_env.bat
+    buildArch: x64
+    msbuildArch: amd64
+    msbuildPlatform: x64
+    CUDA_VERSION: '10.0'
+    buildparameter: --msvc_toolset 14.16 --use_cuda --cuda_version=$(CUDA_VERSION) --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v$(CUDA_VERSION)" --cudnn_home="C:\local\cudnn-$(CUDA_VERSION)-windows10-x64-v7.3.1.20\cuda"
   steps:
+    - task: UsePythonVersion@0
+      inputs: 
+        versionSpec: '3.7' 
+        addToPath: true 
+        architecture: $(buildArch)
+
+    - task: BatchScript@1
+      displayName: 'setup env'
+      inputs:
+        filename: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\$(EnvSetupScript)'
+        modifyEnvironment: true
+        workingFolder: '$(Build.BinariesDirectory)'
+
+    - script: |
+       python -m pip install -q pyopenssl setuptools wheel numpy     
+      workingDirectory: '$(Build.BinariesDirectory)'
+      displayName: 'Install python modules' 
+    
     - template: templates/set-test-data-variables-step.yml
     - template: templates/set-version-number-variables-step.yml
+    - task: PythonScript@0
+      displayName: 'Download test data'
+      inputs:
+        scriptPath: '$(Build.SourcesDirectory)\tools\ci_build\github\download_test_data.py'
+        arguments: --test_data_url $(TestDataUrl) --build_dir $(Build.BinariesDirectory)
+        workingDirectory: $(Build.BinariesDirectory) 
 
-    - template: templates/windows-build-tools-setup-steps.yml
-      parameters:
-        EnvSetupScript: 'setup_env.bat'
-        buildArch: 'amd64'  # amd64 is needed for vcvars target arch
-        setVcvars: true
 
-    - task: PowerShell@1
-      displayName: 'Set CUDA path'
+    - task: PythonScript@0
+      displayName: 'Generate cmake config'
       inputs:
-        scriptName: 'tools/ci_build/github/windows/set_cuda_path.ps1'
```

Comment:
This was previously to allow different versions of CUDA builds. Does this mean only CUDA 10 will be supported going forward? If a build for CUDA 9 is required, can that still be done easily?

Reply:
If we need to support a new version, just change the build args from "--cuda_version=$(CUDA_VERSION) --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v$(CUDA_VERSION) --cudnn_home=xxx" to something new. 

>  If a build for CUDA 9 is required, can that still be done easily?
I'm not sure if our code still compiles. Anyway, it should be very easy to support 10.1 and 10.2. 

The 'set_cuda_path.ps1' script was there because we didn't 'install' CUDA on the machines, we only copies it.  Now I put it into the VM image.  So it's not needed. We don't need to run it every time for every build.  And it's better to not change the system software during the build.

However, I haven't installed CUDA 10.1 and 10.2 on these machines yet.  It's easily doable.